### PR TITLE
Add the option of creating linkable line numbers

### PR DESCRIPTION
**Pull request by aleb**

### CODE REVIEW

**Comment by aleb** [comment]
It would be used like this: https://github.com/daedaleanai/reqtraq/pull/25

File: README.md [hunk 1]
```diff
@@ -172,17 +172,21 @@ err := formatter.Format(w, style, iterator)
 <a id="markdown-the-html-formatter" name="the-html-formatter"></a>
 ### The HTML formatter
 
-By default the `html` registered formatter generates standalone HTML with
```
Comment (alecthomas) [inline README.md]: Please revert unnecessary grammatical changes.

File: formatters/html/html.go [hunk 2]
```diff
@@ -253,6 +264,13 @@ func (f *Formatter) writeHTML(w io.Writer, style *chroma.Style, tokens []chroma.
 	return nil
 }
 
+func (f *Formatter) lineIDAttribute(line int) string {
```
Comment (alecthomas) [inline formatters/html/html.go]: Nice!

**Comment by alecthomas** [comment]
Nice!

This also closes #132.

File: README.md (hunk 1 above)
Comment (aleb) [reply README.md]: You mean README.md should have just one extra line about the new option?

Comment (alecthomas) [reply README.md]: Yes please.

**Comment by aleb** [comment]
done

**Comment by aleb** [comment]
I renamed the option, removing the `With` prefix!

**Comment by alecthomas** [comment]
ThAnks!

**Pull request merged**